README.md

```diff
@@ -82,7 +82,7 @@ MACHINES=[{"name": "My Machine", "type": "manual", "ip": "1.2.3.4", "username":
 - awsSecretAccessKey AWS private key
 - awsKeyName private key name
 - awsPrivateKey (Defaults to /opt/back/id_rsa) path to where the key will be stored
-- awsImage (Defaults to ami-09e61366) Nanocloud's execution servers default image)
+- awsImage (Defaults to empty string) Nanocloud's execution servers default image
```

Comment:
There is no default value if the default value is expected to be set by the administrator.